exercises/practice/mazy-mice/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+## Instructions
```

Comment:
The convention is to start headers in Markdown files with `#`, not `##`, see https://exercism.org/docs/building/markdown/markdown#h-headings

Reply:
fixed